src/gwt/src/org/rstudio/core/rebind/command/ShortcutsEmitter.java

```diff
@@ -206,11 +203,6 @@ else if (keys.size() == 2)
          throw new UnableToCompleteException();
       }
       
-      if (handlesEventPropagation.equals("true"))
-      {
-         writer.println("ShortcutManager.INSTANCE.registerCommandHandlesEventPropagation(" + command + ");");
```

Comment:
It looks like this is the only caller of `registerCommandHandlesEventPropagation`. Can we remove that method, too (and any other now-dead infrastructure?)

Reply:
Yes -- I'll take that out.